src/components/NotFoundPage/NotFoundPage.js

```diff
@@ -6,8 +6,8 @@ import "./NotFoundPage.scss";
 const NotFoundPage = () => {
   return (
     <div className="not-found-page">
-      <h1>Page Not Found</h1>
-      <h2>404</h2>
+      <h1 className="not-found-page__header">Page Not Found</h1>
+      <h2 className="not-found-page__header">404</h2>
```

Comment:
From the semantic view this looks very strange, please use different conventions such one below:

```suggestion
      <h1>
        Page Not Found <span>404</span> {/* and display span as block */}
      </h1>
```